fivem_script/tokovoip_script/src/c_utils.lua

```diff
@@ -182,9 +182,9 @@ end
 function getPlayers()
 	local players = {};
 
-	for i = -1,256 do
+	for i = 0, 256 do
 		if (NetworkIsPlayerActive(i)) then
-			players[#players + 1] = i;
+			table.insert(players, i);
```

Comment:
```suggestion
			players[#players + 1] = i;
```

This should be swapped back to this because it is more performant. You can see the performance differences here: https://springrts.com/wiki/Lua_Performance#TEST_12:_Adding_Table_Items_.28table.insert_vs._.5B_.5D.29

Reply:
I thought so yeah, not sure why it was changed in the PR. Thanks

Reply:
if you are worried about tiny optimization
you should also put almost everything as local
function & natives